nss-cohort-37/Trestlebridge-Silver-Snakes
# Wggbranch

Branch:

- wggbranch

Feature:

- Add duck houses to farm and display list of them in farm status

Testing instructions:

1. Go to create facility
2. Add duck house
3. Go to farm status
4. Confirm duck house listing
